core_lib/src/interface/editor.cpp

```diff
@@ -508,108 +502,193 @@ void Editor::resetAutoSaveCounter()
     mAutosaveCounter = 0;
 }
 
-void Editor::cut()
+void Editor::copy()
+{
+    Layer* currentLayer = layers()->currentLayer();
+
+    Q_ASSERT(currentLayer != nullptr);
+
+    bool canCopy = clipboards()->canCopy(mFrame, currentLayer);
+
+    if (!canCopy) { return; }
+
+    backup("Copy");
```

Comment:
The action should be possible perform to trigger the menu highlight, so that the user can see that they activated the shortcut, otherwise they might become confused or infuriating at the keyboard for seemingly not doing what the user asks of it. Whether anything happens though, that's a different matter if you ask me.

I will fix the string